app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -70,11 +71,11 @@ protected function create(array $data)
             'username' => $data['name'],
             'password' => Hash::make($data['password']),
             'address' => $data['address'] ?? '',
-            'phone' => $data['phone'] ?? '',
+            'phone' => $data['phone'],
```

Comment:
Khi đăng ký không cần số điện thoại

Reply:
khi login co the login bang phone hoac email. nen khi khoi tao e van phai co phone

Reply:
Anh đang yêu cầu login bằng username hoặc email